src/locale/messages.fr.xlf

```diff
@@ -1,313 +1,365 @@
-        <target state="new">Notice - destiné à informer la personne en charge de la zone, ne correspond pas nécessairement à un problème de configuration</target>
+      <trans-unit datatype="html" id="e863ca91edf6eb602f218bd8e04f9c06a5937e95">
+        <source>Notice - means something that should be known by the zone's administrator but that need not necessarily be a problem at all.</source>
+        <target state="translated">Note - destiné à informer la personne en charge de la zone, ne correspond pas nécessairement à un problème de configuration.</target>
         <note priority="1" from="description">tooltip notice</note>
       </trans-unit>
       <trans-unit datatype="html" id="5f8e2137b25e61faf62e17cea5f550a5ae2fbf9c">
         <source>Warning - means something that will under some circumstances be a problem, but that is unlikely to be noticed by a casual user.</source>
-        <target state="new">Warning - destiné aux erreurs détectées dans certains cas d&apos;usage spécifiques</target>
+        <target state="translated">Avertissement - destiné aux erreurs détectées dans certains cas d'usage spécifiques.</target>
         <note priority="1" from="description">tooltip warning</note>
       </trans-unit>
-      <trans-unit datatype="html" id="326827985321290994">
-        <source>There is no delegation for the zone</source>
-        <target state="new">Aucune délégation trouvée pour cette zone</target>
-      </trans-unit>
       <trans-unit datatype="html" id="51c0cd045c7c68c5936905244f1a0a9c7d294ed3">
         <source>What is an undelegated domain test?</source>
-        <target state="new">C&apos;est quoi un test sur un nom de domaine non délégué?</target>
+        <target state="translated">Qu'est-ce qu'un test sur un nom de domaine non délégué ?</target>
       </trans-unit>
-      <trans-unit datatype="html" id="2354337809748099123">
-        <source>Zonemaster Backend is not available</source>
-        <target state="new">Zonemaster Backend n&apos;est pas disponible</target>
+      <trans-unit datatype="html" id="7221366950887162430">
+        <source>Zonemaster Backend is not available.</source>
+        <target state="translated">Zonemaster Backend n'est pas disponible.</target>
       </trans-unit>
       <trans-unit id="7b165859f6b561923cffdcf931fb3e8ec4cd755b" datatype="html">
         <source>Copyright © AFNIC and The Swedish Internet Foundation. All Rights Reserved</source>
-        <target state="new">Copyright © AFNIC and The Swedish Internet Foundation. All Rights Reserved</target>
-      </trans-unit>
-      <trans-unit id="4e59adf0c565d3c2601c3128d0c94a3aa44513f4" datatype="html">
-        <source>Delegation Signers (DS records)</source>
-        <target state="new">Delegation Signers (DS records)</target>
+        <target state="translated">Copyright © AFNIC and The Swedish Internet Foundation. All Rights Reserved</target>
```

Comment:
Translate as: “Copyright © AFNIC et The Swedish Internet Foundation. Tous droits réservés”.

Reply:
Done